### ISSUE TITLE
Please post ramps 1.6 plus

### ISSUE THREAD

**Issue by drmulligan** [bigtreetech/ramps-1.6#3]
bigtreetech is now selling the ramps 1.6 plus with integrated SPI lines for tmc2130.

Please be aware that the SPI lines for the TMC2130 does not work with the Re-Arm board as D40, D42, D65 and D66 are missing.  The situation is worse if you are using a RRD Full Graphics LCD as it doesn't share SPI lines well, or at least in Marlin 2.0 this isn't working and there is a sanity check preventing the configuration from even compiling.   I would love to know how to disconnect MOSI, MISO and SCK from the driver sockets so that I only have to plug them into one of the driver sockets.  I still want those 3 lines to go to AUX3 for the LCD though.

**Comment by olioli2345** [comment]
@drmulligan - have you managed to get anything working with the 1.6+? I, too, have purchase it from China, yet there is no documentation or guidance on setting up Marlin anywhere.

**Comment by ax00b3** [comment]
I didn't understand anything from Alex's video, is there a step by step guide?

**Comment by EsserPrototyping** [comment]
Can’t help with the display, but I’ve managed to get rearm and ramps 1.6 plus with tmc2130 running with spi. I have disconnected the CS lines by removing the jumpers, defined them on other free pins and connected them via jumper cables. In marlin i have defined software spi, but giving the software pins the same as the original.. (D50, D51, D52 ... translated to the rearm board.. i don’t know them right know)
I can’t tell if this is the smart way, but i think the compiler will understand that redefining hardware spi pins as software spi pins is useless and use them as normal. However, onboard spi without jumper cables for mosi, miso and sck is working with this setting. Maybe this problem is solved since it’s not the latest version of marlin i am using.

**Comment by drmulligan** [comment]
@EsserPrototyping I assume this means you do not have a RRD LCD display then.  There are two problems with RE-ARM and this board, one you've addressed with missing lines used for CS.  The other is MOSI, MOSO and SCK cannot be shared with the LCD.  The problem is RRD LCD boards use a controller which does not honour it's CS signal chatting on the other 3 when it's not supposed to.

**Comment by EsserPrototyping** [comment]
Yes, i don´t use a LCD. I´m using this board for a pick and place machine.. but after installing the skr 1.3 in my printer, i think you can have a look at the pin mapping of the skr 1.3, because it´s the same cpu, there is maybe a way to copy the spi pin mapping and use jumper wires for spi to the drivers too.
In general, replacing the re-arm + ramps combo with the skr 1.3 is probably the better solution if the LCD is working with it. (The time it will take to make the ramps work is maybe more expensive than ordering the new board)
I´m swithing to the skr pro with my pick and place, because i need 6 axies.
